ClinGen/clincoded
# Connects to #769. Computational eval forms. Collapsible groups.

* Computational eval forms are in
* Collapsible panels for both population and computational tabs
* CSS updated for wider content area in tabs
* CSS updated so that tabs and panel css names do not collide
* `form.js` updated to remove form title and Panel groups; these were moved to their respective tabs

Population tab w/ Interpretation:
![image](https://cloud.githubusercontent.com/assets/4326866/16825070/378684fc-4926-11e6-9c7e-deed6f711817.png)
Predictors tab w/ Interpretation:
![image](https://cloud.githubusercontent.com/assets/4326866/16825077/447f1e76-4926-11e6-849e-39d62e9ec458.png)

Testing:

1. Get to VCI, w/ no interpretation
2. Check population tab, check that panel displays and collapses properly
3. Check predictors tab, check that panels display and collapse properly
4. Add interpretation
5. Check population tab, check that form + data displays and collapses properly
6. Check predictors tab, check that forms + data displays and collapses properly
7. Check evaluation forms and output data in predictors tab
  * If CRITERIA1 OR CRITERIA2 criteria, make sure that both criteria cannot be checked together
  * Make sure that criteria that are disease dependent are disabled while no disease is associated
  * Make sure when saving, the feedback message reports back the proper criteria codes
  * When saving, only the first set of criteria code should actually save evidence data, the rest of the forms should have no evidence data associated with them
